src/dmd/typesem.d

```diff
@@ -237,9 +244,26 @@ private void resolveHelper(TypeQualified mt, const ref Loc loc, Scope* sc, Dsymb
                 *pt = Type.terror;
                 return;
             }
+
+            void helper3()
```

Comment:
I suggest using a more descriptive name to help self-document the code.  I would suggest a name, but I'm afraid I can't decipher what it's doing, further motivating the need for a descriptive name.

Reply:
I don't really know what it's doing, either.

Reply:
I just try to leave things better than I found them.

Reply:
> I don't really know what it's doing, either.

At first I laughed, but now I realize that's not funny.

Reply:
`resolveHelper` is also a most unhelpful name. But I wrote the darn thing 20 years ago, and it'll take more work than this PR to make it a virtuous piece of code.

Reply:
:)

Reply:
`symbolToExpression` maybe?

Reply:
That's not at all what it does.